src/tools/segmentation/FreehandScissorsTool.js

```diff
@@ -44,4 +68,21 @@ export default class FreehandScissorsTool extends BaseTool {
 
     super(props, defaultProps);
   }
+
+  setLabelmapChangeCallback(labelmapChangeCallback) {
+    this.labelmapChangeCallback = labelmapChangeCallback;
+  }
+}
+
+function callStrategyWithLabelmapEvent(
```

Comment:
Why not make this function a part of the class so you dont have to also pass `labelmapChangeCallback`?

Reply:
When I pass it to the class, I get an error saying can't call this before super(). Although it doesn't really make sense because I'm calling this when I pass in the labelmap callback as well

Reply:
Actually nevermind ignore this

Reply:
e51fbe9002c661625281d1dece3980a2f98967a6